diff --git a/includes/HomepageModules/SuggestedEdits.php b/includes/HomepageModules/SuggestedEdits.php
index 14231727b..f282ceeda 100644
--- a/includes/HomepageModules/SuggestedEdits.php
+++ b/includes/HomepageModules/SuggestedEdits.php
@@ -275,8 +275,7 @@ class SuggestedEdits extends BaseModule {
 		UserOptionsLookup $userOptionsLookup
 	) {
 		return self::isEnabledForAnyone( $context->getConfig() ) &&
-			$context->getConfig()->get( 'GEHomepageSuggestedEditsEnableTopics' ) &&
-			ExtensionRegistry::getInstance()->isLoaded( 'WikimediaMessages' );
+			$context->getConfig()->get( 'GEHomepageSuggestedEditsEnableTopics' );
 	}
 
 	/**
